content/assignments/hello-world/auto-grading-test.md

```diff
@@ -0,0 +1,87 @@
+      public class AutogradingTests
+      {
+         [TestMethod]
+         public void TestToDemonstrateWhatATestMethodLooksLike()
+         {
+            //test code here
+         }
+
+         [TestMethod]
+         public void SecondTestToDemonstrateWhatATestMethodLooksLike()
+         {
+            //test code here
+         }
+      }
+   ```
+1. Run all of your tests using the Test Explorer window.
+   1. Mac Users: 
+      1. **View** > **Tests**  
+      1. Select the green triangle on the left of the Test Explorer Window. 
+      1. In the Test Explorer, step through the file tree until you find the failed test. 
+      1. If you need more help running the tests, checkout this [guide](https://learn.microsoft.com/en-us/visualstudio/mac/testing?view=vsmac-2022#running-tests) for running tests in Visual Studio for Mac
+   1. Windows Users 
+      1. **Test** > **Windows** > **Test Explorer**
+      1. Select the `Run All` icon to run all of your tests. 
+      1. In the Test Explorer, step through your file tree until you find the failed test.
+      1. If you need more help running the tests, checkout this [guide](https://learn.microsoft.com/en-us/visualstudio/test/run-unit-tests-with-test-explorer?view=vs-2022) for running tests in Visual Studio.
+
+
+1. Visual Studio opens a Test Results Pane every time it runs a test. This pane provides you with detailed information about which tests passed and which tests failed.  The pane will also provide you with output for any failed tests, including what was expected and what was actually received.  This is a great place to look when dealing with a failing test.
+
+   Now that you have run the tests, you should see that one of the tests has failed and the other passed. We have not touched any code yet. Use the Test Results Pane to navigate into which specific test has failed.
+
+   In this case, the `TestIfSayHelloReturnsCorrectValue()` test, expected our code to output `Hello, World!` Instead, the method is returning is `&&&` causing the test to fail. Be sure to explore your test output. The output will show you what was expected and what was actually received.  Often messages will be provided for a failed test. In this test, the message is "Incorrect message displayed".
+
+   The `TestSayHelloReturnTypeIsString()` test passed. This should pass because the `SayHello()` method is a string method. You can verify this in line 8 of the `SayHelloClass.cs`. Use this test to help you explore the Test Results Pane and testing process. 
+
+1. Try running each test individually before moving on to the next step. You can select individual tests by right-clicking on the test name either directly in the test class or the Test Explorer window.
+
+   You will still have the same results as before, but now you have the skills to run tests all together or one at a time. This is something that may come in handy as you work on larger assignments that may have many tests.
+
+
+### Next Steps
```

Comment:
I'm not a fan of the Next Steps at the bottom of each page.  Could just be a sentence to move to the next page.